app/models/validations/shared_validations.rb

```diff
@@ -68,4 +68,15 @@ def inactive_status(date, resource)
 
     { scope: status, date: date&.to_formatted_s(:govuk_date), deactivation_date: closest_reactivation&.deactivation_date&.to_formatted_s(:govuk_date) }
   end
+
+  def validate_valid_radio_option(record)
```

Comment:
Do you need to rebase so that the feature flag is here as well?

Reply:
that's done